core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -265,23 +292,84 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
       threadId);
     const threadMessageBackendDicts = response.messages;
     this.reviewMessage = threadMessageBackendDicts.map(
-      m => ThreadMessage.createFromBackendDict(m))[1].text;
+      m => ThreadMessage.createFromBackendDict(m))[1]?.text ?? '';
   }
 
-  showNextItemToReview(suggestionId: string): void {
-    this.resolvedSuggestionIds.push(this.activeSuggestionId);
-    if (this.lastSuggestionToReview) {
+  gotoNextItem(): void {
```

Comment:
Can you explain why is this changed just because we are introducing this on mobile?

Reply:
This change is made to introduce navigation in the modal (on both desktop and mobile). 
![image](https://user-images.githubusercontent.com/73544247/177786501-388968e6-bb1c-496c-a857-d640384a3756.png)

Currently the only way to navigate forward (from within the modal) is to review the contents. This function allows users to navigate without having to actually review anything.